certif/tests/unit/routes/authenticated/sessions/import_test.js

```diff
@@ -0,0 +1,130 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import sinon from 'sinon';
+import Service from '@ember/service';
+
+module('Unit | Route | authenticated/sessions/import', function (hooks) {
+  setupTest(hooks);
+
+  module('when top level domain is org', function () {
+    module('when current language is french', function () {
+      test('it should not redirect to sessions list page', async function (assert) {
```

Comment:
C'est du détail mais peut etre préciser dans le it "it should not redirect to sessions list page, when accessing to import page" (avec un meilleur anglais 👀 ) ?
Apres on comprend en lisant le test mais au cas où :)

Reply:
je rajoute ça !